docs/embedding/static-embedding-parameters.md

```diff
@@ -180,21 +180,31 @@ your_embedding_url#theme=night&bordered=false&titled=true
 
 You can preview appearance settings from your question or dashboard's embedded appearance settings.
 
-| Parameter name             | Possible values                               |
-| -------------------------- | --------------------------------------------- |
-| `bordered`                 | true, false                                   |
-| `titled`                   | true, false                                   |
-| `theme`                    | null (default), night                         |
-| `refresh` (dashboard only) | integer (seconds, e.g., `refresh=60`)         |
-| `font`\*                   | [font name](../configuring-metabase/fonts.md) |
-| `downloads`\*\*            | true, false                                   |
+| Parameter name             | Possible values                                                                                                                                |
+| -------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------- |
+| `background`               | true (default), false. Dashboards only.                                                                                                        |
+| `bordered`                 | true (default), false.                                                                                                                         |
+| `titled`                   | true (default), false.                                                                                                                         |
+| `theme`                    | null (default), night. `theme=transparent` should work, but is deprecated (see [Transparent backgrounds](#transparent-backgrounds-for-embeds)) |
+| `refresh` (dashboard only) | integer (seconds, e.g., `refresh=60`).                                                                                                         |
+| `font`\*                   | [font name](../configuring-metabase/fonts.md)                                                                                                  |
+| `downloads`\*\*            | true (default), false.                                                                                                                         |
 
 \* Avalable on [Pro](https://www.metabase.com/product/pro) and [Enterprise](https://www.metabase.com/product/enterprise) plans
 
 \*\* Disabling downloads is available on [Pro](https://www.metabase.com/product/pro) and [Enterprise](https://www.metabase.com/product/enterprise) plans.
 
 For global appearance settings, such as the colors and fonts used across your entire Metabase instance, see [Customizing Metabase's appearance](../configuring-metabase/appearance.md).
 
+## Transparent backgrounds for embeds
+
+Making an embed transparent depends on the type of embed:
+
+- Dashboards: set `background=false`.
```

Comment:
Maybe we could add a note that this can be combined with default or night theme? Or is it obvious?